public/assets/css/style.css

```diff
@@ -3364,12 +3364,51 @@ hr {
   -moz-transition: 0.4s;
   -o-transition: 0.4s;
   transition: 0.4s;
+  /* width: 97%; */
+  margin: auto;
+}
+.courses-area .properties__img{
+    width: 80%;
+    height:33rem ;
+    object-fit: fill;
+
+}
+.courses-area .properties__link{
+  color: #4255a4;
+  text-Decoration: none;
+  text-align: left;
+  font-weight: 500;
+  font-Size:2rem;
+}
+.courses-area .properties__link:hover{
+  color: #4255a4;
+  text-Decoration: underline;
+  font-Size:2rem;
+}
+.courses-area .properties__info{
+  max-height:13rem; 
+  overflow-y: hidden;
+  text-overflow: ellipsis;
```

Comment:
move all css of internship card in a css file named intership-section
and please update css names from courses -> internship
we can have css file in `styles` folder